spec/features/idv/account_creation_spec.rb

```diff
@@ -8,4 +8,9 @@
     it_behaves_like 'idv account creation', :saml
     it_behaves_like 'idv account creation', :oidc
   end
+
+  context 'signing in when profile is pending USPS verification' do
+    it_behaves_like 'signing in with pending USPS verification', :saml
+    it_behaves_like 'signing in with pending USPS verification', :oidc
+  end
```

Comment:
I just realized this does not involve account creation. I will move to a different folder.